Add `FinalFormFileSelect` (#1461)
vivid-planet/comet: .changeset/strong-onions-cough.md

> ---
> "@comet/admin": major
> ---
> 
> Add the `FinalFormFileSelect` component to allow picking files and using drag-and-drop to upload files

Reviewer: ```suggestion
---
"@comet/admin": minor
---

Add `FinalFormFileSelect` component

Allows selecting files via the file dialog or using drag-and-drop.
```
